sync/metrics.go

```diff
@@ -71,6 +72,16 @@ func newMetrics() (*metrics, error) {
 		return nil, err
 	}
 
+	failedAgainstSubjHead, err := meter.Int64Counter(
```

Comment:
IMO I feel like we'd need more context here to make this metric valuable (e.g. when it failed, what heights, etc)
Bc just the number by itself won't be really helpful to diagnose where / how the problem occurred.

I'm in favour of leaving this metric out for now and adding it back later in a meaningful way. This really rarely happens anyway

Reply:
So...adding attributes like `height` will make it more valuable?

Also, `e.g. when it failed` what exactly? I assume you're not talking about time when it happened but the place? Probably this should be reflected in metric's name, do you have a better naming for this?

Reply:
You'd need to know: 

height/hash of subjective head when this occurred, + height/hash of failed network head and probably timestamp as well

Reply:
Done